examples/Notebook/Custom Keyboard Shortcuts.ipynb

```diff
@@ -38,6 +38,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "The keypress `r` in command mode is by default bound to change the type of a cell ro `raw`. This action will thus be unavaillable from the keyboard now."
```

Comment:
cell **to** `raw`

Also, I think moving things around in the sentence makes it read better:
"By default the keypress `r`, while in command mode, changes the type of the selected cell to `raw`.  This shortcut is overridden by the code in the previous cell, and thus the action no longer be available via the keypress `r`."